ibraheemosule/skoolizy
# za - style sub route

# Why this PR?
added styling to the sub route

# What does this PR do?
a border is added to separate the sub route.

# How to test your work?
Check the sub routes in the Classroom.tsx page you should see a border separating the routes.

# Pages to see your changes?
Classroom/record-grade/Record Grade.tsx
